tests/puppeteer/pages/BO/catalog/files/index.js

```diff
@@ -148,8 +150,6 @@ module.exports = class Files extends BOBasePage {
    * @return {Promise<textContent>}
```

Comment:
```suggestion
   * @return {Promise<string>}
```

Reply:
Sorry, didn't see that, i will fix it tomorrow. 
We merged it today because test will fail otherwise.